zerver/lib/types.py

```diff
@@ -281,3 +281,4 @@ class GroupPermissionSetting:
     allow_owners_group: bool
     allow_nobody_group: bool
     default_group_name: str
+    default_for_system_groups: Optional[str] = None
```

Comment:
Have added this since I guess we might have settings in future that might need different default for system groups. We would have the same default for all groups atleast for this setting once we solve #26148, but then also we might have different default for single user groups.

Might want to rename this or add a new field then in future depending upon what further group-level settings we add. 
Or we can probably skip this for now as we only have one setting currently, not sure.

Have kept this optional since this will not be used for stream and realm setings. Though we might want to have different default for public and private streams but that can be handled in future.

Reply:
Yeah, this seems pretty reasonable for now.